File: SDKs/Aspose.Pdf_Cloud_SDK_For_Java/src/main/java/com/aspose/pdf/model/DocumentTextReplaceResponse.java
package com.aspose.pdf.model;

import com.aspose.pdf.model.Document;
public class DocumentTextReplaceResponse {
  private Document Document = null;
  private Integer Matches = null;
  private String Code = null;
  private String Status = null;
  /**
	 * getDocument
	 * Gets Document
	 * @return Document
	 */
  public Document getDocument() {
    return Document;
  }

	/**
	 * setDocument
	 * Sets Document
	 * @param Document Document
	 */
  public void setDocument(Document Document) {
    this.Document = Document;
  }

  /**
	 * getMatches
	 * Gets Integer
	 * @return Matches
	 */
  public Integer getMatches() {
    return Matches;
  }

	/**
	 * setMatches
	 * Sets Integer
	 * @param Matches Integer
	 */
  public void setMatches(Integer Matches) {
    this.Matches = Matches;
  }

  /**
	 * getCode
	 * Gets String
	 * @return Code
	 */
  public String getCode() {
    return Code;
  }

	/**
	 * setCode
	 * Sets String
	 * @param Code String
	 */
  public void setCode(String Code) {
    this.Code = Code;
  }

  /**
	 * getStatus
	 * Gets String
	 * @return Status
	 */
  public String getStatus() {
    return Status;
  }

	/**
	 * setStatus
	 * Sets String
	 * @param Status String
	 */
  public void setStatus(String Status) {
    this.Status = Status;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class DocumentTextReplaceResponse {\n");
    sb.append("  Document: ").append(Document).append("\n");
    sb.append("  Matches: ").append(Matches).append("\n");
    sb.append("  Code: ").append(Code).append("\n");
    sb.append("  Status: ").append(Status).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: SDKs/Aspose.Pdf_Cloud_SDK_for_CSharp/src/Com/Aspose/PDF/Model/DocumentTextReplaceResponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.PDF.Model {
  public class DocumentTextReplaceResponse {
    public Document Document { get; set; }

    public int? Matches { get; set; }

    public string Code { get; set; }

    public string Status { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class DocumentTextReplaceResponse {\n");
      sb.Append("  Document: ").Append(Document).Append("\n");
      sb.Append("  Matches: ").Append(Matches).Append("\n");
      sb.Append("  Code: ").Append(Code).Append("\n");
      sb.Append("  Status: ").Append(Status).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }
